lib/spack/spack/cmd/create.py

```diff
@@ -568,7 +583,7 @@ def create(parser, args):
     url = get_url(args)
     versions, guesser = get_versions(args, name)
     build_system = get_build_system(args, guesser)
-
+              
```

Comment:
Look, Python is a language in which whitespace means things. More whitespace = more meaning. That's just math.

(More seriously I'll fix it)